src/sound_type.h

```diff
@@ -11,7 +11,7 @@
 #define SOUND_TYPE_H
 
 struct SoundEntry {
-	uint8 file_slot;
+	class RandomAccessFile *file;
```

Comment:
Given how "small" the RAF header is, possibly fine to just #include it? Or maybe not necessary, idk

Reply:
I could, but that in turn pull in fileio_type.h making it > 200 lines. Not to mention <string>, although that might already be included somewhere else too.